@here/harp-geoutils/lib/projection/MercatorProjection.ts

```diff
@@ -183,9 +183,131 @@ export class MercatorProjection extends Projection {
         }
         return normal;
     }
+
+    reprojectPoint(
+        sourceProjection: Projection,
+        worldPos: Vector3Like,
+        result?: Vector3Like
+    ): Vector3Like {
+        // this implementation of [[reprojectPoint]] supports both
+        // [[WebMercatorProjection]] and [[MercatorProjection]]. The only
+        // difference betweeen these two variants of WEB Mercator
+        // is in the orientation of the Y axis, so we just flip Y coordinates
+        // when reprojecting between them.
+        if (
+            sourceProjection !== this &&
+            (sourceProjection === webMercatorProjection || sourceProjection === mercatorProjection)
```

Comment:
If source is mercator (and this is also mercator) we should not flip the y axis

Reply:
yes, and that is captured by "sourceProjection !== this"